config.go

```diff
@@ -500,6 +502,7 @@ func DefaultConfig() Config {
 		DB:                      lncfg.DefaultDB(),
 		registeredChains:        chainreg.NewChainRegistry(),
 		ActiveNetParams:         chainreg.BitcoinTestNetParams,
+		CoinSelectionStrategy:   "largest",
```

Comment:
nit: should add a constant for this. Or maybe even a `String()` method on the `CoinSelectionStrategy` itself.

Reply:
Added constant. Didn't want to use `String()` and surface that in the command line flags.